src/types/onyx/Fund.ts

```diff
@@ -18,11 +20,12 @@ type AccountData = {
     created?: string;
     currency?: string;
     fundID?: number;
+    bank?: string;
 };
 
 type Fund = {
     accountData?: AccountData;
-    accountType?: string;
+    accountType?: typeof CONST.PAYMENT_METHODS.DEBIT_CARD;
```

Comment:
Is this really correct?

Reply:
@Skalakid Please check it once you are back